### PR TITLE
Adding sensors for trip statistics since refueling

### PR DESCRIPTION
**Pull request by loeffelpan**
As discussed on discord I added sensors for trip statistics since refueling.
API url was `cyclic?newest` at the end.

So this was basically some copy&paste.

### CODE REVIEW

**Comment by loeffelpan** [comment]
In addition I precised the log messages for trip statistics. See https://github.com/lendy007/skodaconnect/pull/69/commits/63d89da95bad23111eb886b8d3d55f612bc55946#diff-d71651901ee84eed15f18f518e8cb094b33c8d536f3844ba7014273239e47dedR1130
And renamed one entity from "lenght" to "distance". Length is fine in the code, but not a entity name. See https://github.com/lendy007/skodaconnect/pull/69/commits/63d89da95bad23111eb886b8d3d55f612bc55946#diff-0dfabf2110152af8a89d1e6eb68e578c54940981ba1208dbcd74aa8f1d06cdd5R1339

File: skodaconnect/connection.py [hunk 1]
```diff
@@ -1093,21 +1093,43 @@ async def getTripStatistics(self, vin):
...
                 return data
             elif short.get('status_code', False):
                 _LOGGER.warning(f'Could not fetch trip statistics, HTTP status code: {short.get("status_code")}')
             elif long.get('status_code', False):
-                _LOGGER.warning(f'Could not fetch trip statistics, HTTP status code: {long.get("status_code")}')
+                _LOGGER.warning(f'Could not fetch longtermin trip statistics, HTTP status code: {long.get("status_code")}')
+            elif cyclic.get('status_code', False):
+                _LOGGER.warning(f'Could not fetch cyclic trip statistics, HTTP status code: {cyclic.get("status_code")}')
             else:
                 _LOGGER.info(f'Unhandled error while trying to fetch trip statistics')
```
Comment (dvx76) [inline skodaconnect/connection.py:1134]: Consider reducing and simplifying this huge if statement to something like:

```python
        data = {}
        try:
            ...
            if short.get('tripData', False):
                data['tripstatistics'] = short.get('tripData', {})
            else:
                _LOGGER.warning(f'Could not fetch trip statistics, HTTP status code: {short.get("status_code", "none")}')
            if long.get('tripData', False):
                data['longtermstatistics'] = long.get('tripData', {})
            else:
                _LOGGER.warning(f'Could not fetch longtermin trip statistics, HTTP status code: {long.get("status_code", "none")}')
            if cyclic.get('tripData', False):
                data['cyclicstatistics'] = cyclic.get('tripData', {})
            else:
                _LOGGER.warning(f'Could not fetch cyclic trip statistics, HTTP status code: {cyclic.get("status_code", "none")}')
        except Exception as error:
            _LOGGER.warning(f'Could not fetch trip statistics, error: {error}')
        return data


```

Comment (loeffelpan) [reply skodaconnect/connection.py:1134]: I get your point. Code could be easier, cleaner and shorter.
But there are dozens of similiar code-blocks with `if .. elif .. elif .. ` and I tried not to change the style of code.
I could make another commit I you like (already prepared).

**Comment by dvx76** [comment]
Thanks. LGTM

**Comment by Farfar** [comment]
Thanks!

**Pull request merged**